### DISCUSSION TITLE
表格处理

### DISCUSSION THREAD

**Discussion by Sep24thLegend** [magicpdf/Magic-PDF#930] [General]
表格的处理速度有什么加速方法嘛，表格处理速度太慢了
![image](https://github.com/user-attachments/assets/a2c9635b-e4ed-4800-a266-2063822c9668)

**Comment by DayDreamerEric** [comment]
换一个轻量化的模型呗

**Comment by myhloli** [comment]
我们会在下个版本加入rapid_table，在表格解析性能上提升很大，最近就会发版支持。

**Reply by Sep24thLegend** [reply]
好的，感谢
